test/pkg/environment/aws/environment.go

```diff
@@ -72,6 +75,7 @@ type Environment struct {
 
 	ClusterName       string
 	ClusterEndpoint   string
+	ClusterVersion    string
```

Comment:
Does using `env.K8sVersion()` not work for these tests?

Reply:
Yeah, that would work here. Didn't see that